packages/server/src/main/java/software/uncharted/terarium/hmiserver/utils/rebac/ReBACService.java

```diff
@@ -313,26 +313,19 @@ public PermissionGroup getGroup(String id) {
 		return permissionGroup;
 	}
 
-	public boolean canRead(SchemaObject who, SchemaObject what) throws Exception {
+	public boolean can(SchemaObject who, Schema.Permission permission, SchemaObject what) throws Exception {
```

Comment:
Could those method have Java Doc to understand how to read and use them?
It would be easier to use.

Reply:
done